.github/workflows/windows.yml

```diff
@@ -1,17 +1,14 @@
 name: Windows (VS 2019, Python 3.11)
 on:
   workflow_dispatch:
-  schedule:
-    # at 00:00 on workdays
-    - cron: '0 0 * * 1,2,3,4,5'
```

Comment:
I do not think it is needed as the post-commit is enabled and the self-hosted runners are used.